web/templates/settings_overlay.hbs

```diff
@@ -36,7 +36,7 @@
                     {{/unless}}
                     <li class="sidebar-item" tabindex="0" data-section="alert-words">
                         <i class="sidebar-item-icon fa fa-book" aria-hidden="true"></i>
-                        <div class="text">{{t "Alert words" }}</div>
+                        <div class="text">{{t "Watched phrases" }}</div>
```

Comment:
Why is this capitalized in the screenshot?

Reply:
Ahh, I forgot to update the screenshot. I’ve now updated the screenshot.